commands/drawCommand.js

```diff
@@ -22,14 +25,101 @@ export const RARITIES = [
   "Gold Ultra Rare",
 ];
 
+const storingDraws = async (message) => {
```

Comment:
methods should be in simple present tense, not continuous present tense (so `storeDraws` instead of `storingDraws`)

Reply:
actually a more accurate name for what this does is `drawAndStore`